{"name": "test_trigger_test_button", "status": "failed", "statusDetails": {"message": "AssertionError: Failed\nassert 1 == 2", "trace": "def test_trigger_test_button():\n        print(\"Testing Trigger Test Button\")\n    \n>       assert 1==2, \"Failed\"\nE       AssertionError: Failed\nE       assert 1 == 2\n\ntests\\test_stream.py:6: AssertionError"}, "attachments": [{"name": "stdout", "source": "9a92498d-2c01-43d9-b77b-36e0c5b73f09-attachment.txt", "type": "text/plain"}], "start": 1740428313895, "stop": 1740428313895, "uuid": "fc479322-2540-45b6-90d5-e7e6799f948c", "historyId": "1f8fbfd19b9570e4ed564d5bd541a9d6", "testCaseId": "1f8fbfd19b9570e4ed564d5bd541a9d6", "fullName": "tests.test_stream#test_trigger_test_button", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_stream"}, {"name": "host", "value": "DESKTOP-LUCDFB2"}, {"name": "thread", "value": "35020-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_stream"}]}